Apps/phironnet/ironnet.json

```diff
@@ -0,0 +1,512 @@
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "report bad activity",
+            "identifier": "irondefense_report_observed_bad_activity",
+            "description": "Reports observed bad activity to IronDefense",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "name": {
+                    "description": "Identifying name of the observed bad activity",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "order": 0
+                },
+                "description": {
+                    "description": "Description of the observed bad activity",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "order": 1
+                },
+                "domain": {
+                    "description": "Domain associated with the observed bad activity",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "contains": ["domain"],
+                    "order": 2
+                },
+                "ip": {
+                    "description": "IP associated with the observed bad activity",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "contains": ["ip"],
+                    "order": 3
+                },
+                "activity_start_time": {
+                    "description": "Start time of observed bad activity - in RFC3339 format (https://tools.ietf.org/html/rfc3339)",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 4
+                },
+                "activity_end_time": {
+                    "description": "End time of observed bad activity - in RFC3339 format (https://tools.ietf.org/html/rfc3339)",
+                    "data_type": "string",
+                    "required": false,
+                    "order": 5
+                }
+            },
+            "output": [
```

Comment:
Optionally for each `output` object, you can also add keys `column_name` and `column_order` where `column_name` may be something like `Status`, and `column_order` is `0` and increases by 1 for every column that you add

Reply:
ok - I added column_name/column_orders to the response fields I thought were most relevant.